tests/test_flake8_spellcheck.py

```diff
@@ -116,6 +116,17 @@ def test_empty_comment(self, flake8dir):
         result = flake8dir.run_flake8()
         assert result.out_lines == []
 
+    # Regression test for https://github.com/MichaelAquilina/flake8-spellcheck/issues/36
+    def test_type_and_noqa(self, flake8dir):
+        flake8dir.make_example_py(
+            """
+            # the comment below should not fail on `W503` or `W504`  # noqa: SC100
+            # type: ignore  # noqa: W503  # noqa: W504
```

Comment:
should this be preceeded by something to make sure its a realistic example?

Reply:
good point, thank you, i'll add that!